src/foldermodel.cpp

```diff
@@ -122,7 +122,7 @@ void FolderModel::loadPendingThumbnails() {
     hasPendingThumbnailHandler_ = false;
     for(auto& item: thumbnailData_) {
         if(!item.pendingThumbnails_.empty()) {
-            auto job = new Fm::ThumbnailJob(item.pendingThumbnails_, item.size_);
+            auto job = new Fm::ThumbnailJob(std::move(item.pendingThumbnails_), item.size_);
```

Comment:
> No, item.pendingThumbnails_ is cleared correctly here

Because you know the internal implementation of the `QVector` on which you rely upon here, but...